web/api/web_api_v1.c

```diff
@@ -1215,14 +1215,11 @@ inline int web_client_api_request_v1_info_fill_buffer(RRDHOST *host, BUFFER *wb)
 
 #ifdef ENABLE_ACLK
     buffer_strcat(wb, "\t\"cloud-available\": true,\n");
-    buffer_strcat(wb, "\t\"aclk-ng-available\": true,\n");
-    buffer_strcat(wb, "\t\"aclk-ng-new-cloud-protocol\": true,\n");
-    buffer_strcat(wb, "\t\"aclk-legacy-available\": false,\n");
-    buffer_strcat(wb, "\t\"aclk-implementation\": \"Next Generation\",\n");
+    buffer_strcat(wb, "\t\"aclk-available\": true,\n");
+    buffer_strcat(wb, "\t\"aclk-new-cloud-protocol\": true,\n");
 #else
     buffer_strcat(wb, "\t\"cloud-available\": false,\n");
-    buffer_strcat(wb, "\t\"aclk-ng-available\": false,\n");
-    buffer_strcat(wb, "\t\"aclk-legacy-available\": false,\n");
+    buffer_strcat(wb, "\t\"aclk-available\": false,\n");
```

Comment:
Should we check if backend uses any of those at some point .. ?

Reply:
double checking but i think not. If we are connected to cloud that means we have what was before ACLK-NG otherwise we dont
basically this flag is on the cloud replaced by AgentConnectionUpdated messages which Cloud should consider cannonical

flags in /api/v1/info are more of user/debug thing at this point

Reply:
cloud guys have been notified, no complaints so far
I am therefore assuming no issues with removing this